src/main/java/org/logstash/dissect/fields/AppendField.java

```diff
@@ -0,0 +1,52 @@
+package org.logstash.dissect.fields;
+
+import com.logstash.Event;
+import org.logstash.dissect.ValueResolver;
+
+import java.util.Map;
+
+public final class AppendField extends AbstractField {
+    private static final int ORD = 2;
+
+    private AppendField(String s, int ord) {
+        super(s, ord);
+    }
+
+    public static Field create(String s) {
+        if (hasOrdinal(s)) {
+            String[] parts = s.split("/");
```

Comment:
You rely that you get exactly two elements here. Does the enclosing code ensure that? What happens if I provide `XX/3/5/7`?